util/logtools/make_timeline.ts

```diff
@@ -368,11 +368,11 @@ const assembleTimelineStrings = (
   // If the user entered phase information,
   // process it and store it off.
   const phases: { [name: string]: number } = {};
-  if (args.phase) {
+  if (args.phase !== null && args.phase.length > 0) {
```

Comment:
```suggestion
  if (args.phase !== null) {
```

Length check is unnecessary here as the for loop will do that below.